data/SAMPLE-cat.html

```diff
@@ -1,208 +1,149 @@
 <h2>Catalogus</h2>
-<p class="ednote" title="Over deze catalogus">
+<p class="note" title="Over deze catalogus">
    Deze catalogus is automatisch samengesteld op basis van het UML model 
    "SampleBase" door Imvertor OS 1.26.0.20170711 op July 17, 2017 at 08:06:56.
    <br>
    Wanneer je technische fouten of onvolkomenheden aantreft, geef dit dan door aan <i>arjan@linkedmatter.com</i> en geef de code 
    <i>"green-SampleBase-1.0.0-2-20130318-20170717-090657"</i> door. 
    Voor inhoudelijke fouten neem contact op met het modellenteam: <i>github@basisregistratieondergrond.nl</i>.
    
-</p><a class="anchor" name=""></a><section>
+</p><section>
    <h1>Featuretypes </h1>
-   <a class="anchor" name="global_d6505e1512"></a><section>
+   <section id="global_d6505e1512">
       <h2>FeatureType Bike</h2>
-      
+
       <table>
+         <thead>
+            <colgroup>
+               <col width="30%"></col>
+               <col width="70%"></col>
+            </colgroup>
+         </thead>
```

Comment:
Kolombreedte kan beter worden gespecificeerd in een `<colgroup>` dan individueel per `trow`